client/extensions/woocommerce/state/sites/shipping-zone-locations/actions.js

```diff
@@ -20,7 +20,7 @@ export const fetchShippingZoneLocations = ( siteId, zoneId ) => ( dispatch, getS
 		return;
 	}
 
-	//0 is Rest of the World zone, skip fetching locations
+	//0 is Locations not covered by your other zones zone, skip fetching locations
```

Comment:
@DanReyLop core changed the name of this zone in 3.1. https://github.com/woocommerce/woocommerce/blob/c43afcfbd06b6f929c5f175b42cbd2de44e1c6ef/includes/admin/settings/views/html-admin-page-shipping-zones.php#L26

Shouldn't that change be reflected in Calypso when setting up new sites?